### ISSUE TITLE
Fails install without '.' in @INC

### ISSUE THREAD

**Issue by kentfredric** [adamkennedy/PPI#198]
On 5.25.10 with -Ddefault_inc_excludes_dot
```
Configuring PPI-1.220 ... Can't locate inc/Module/Install.pm in @INC (you may need to install the inc::Module::Install module) (@INC contains: /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at Makefile.PL line 4.
BEGIN failed--compilation aborted at Makefile.PL line 4.
```

**Comment by kentfredric** [comment]
Patching out install defects also leads to test failures:
```
PERL_DL_NONLAZY=1 "/home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/bin/perl5.25.10" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'inc', 'blib/lib', 'blib/arch')" t/*.t
Can't locate t/lib/PPI.pm in @INC (you may need to install the t::lib::PPI module) (@INC contains: /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/07_token.t line 17.
BEGIN failed--compilation aborted at t/07_token.t line 17.
# Looks like your test exited with 2 just after 1.
t/07_token.t ..................... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 446/447 subtests 
Can't locate t/lib/PPI.pm in @INC (you may need to install the t::lib::PPI module) (@INC contains: /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/08_regression.t line 18.
BEGIN failed--compilation aborted at t/08_regression.t line 18.
# Looks like your test exited with 2 just after 1.
t/08_regression.t ................ 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 931/932 subtests 
t/01_compile.t ................... ok
t/03_document.t .................. ok
Can't locate t/lib/PPI.pm in @INC (you may need to install the t::lib::PPI module) (@INC contains: /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/05_lexer.t line 26.
BEGIN failed--compilation aborted at t/05_lexer.t line 26.
# Looks like your test exited with 2 just after 1.
t/05_lexer.t ..................... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 218/219 subtests 
t/10_statement.t ................. ok
t/11_util.t ...................... ok
t/09_normal.t .................... ok
t/14_charsets.t .................. ok
t/16_xml.t ....................... ok
t/13_data.t ...................... ok
t/17_storable.t .................. ok
t/15_transform.t ................. ok
Can't locate t/lib/PPI.pm in @INC (you may need to install the t::lib::PPI module) (@INC contains: /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/19_selftesting.t line 23.
BEGIN failed--compilation aborted at t/19_selftesting.t line 23.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 2 just after 1.
t/19_selftesting.t ............... 
Dubious, test returned 2 (wstat 512, 0x200)
All 1 subtests passed 
t/22_readonly.t .................. ok
t/12_location.t .................. ok
t/18_cache.t ..................... ok
t/20_tokenizer_regression.t ...... ok
Can't locate t/lib/PPI.pm in @INC (you may need to install the t::lib::PPI module) (@INC contains: /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/26_bom.t line 14.
BEGIN failed--compilation aborted at t/26_bom.t line 14.
# Looks like your test exited with 2 just after 1.
t/26_bom.t ....................... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 20/21 subtests 
t/23_file.t ...................... ok
Can't locate t/lib/PPI.pm in @INC (you may need to install the t::lib::PPI module) (@INC contains: /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/inc /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/lib /home/kent/.cpanm/work/1488176942.32397/PPI-1.220/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/25_increment.t line 22.
BEGIN failed--compilation aborted at t/25_increment.t line 22.
# Looks like your test exited with 2 just after 1.
t/25_increment.t ................. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 3875/3876 subtests 
t/27_complete.t .................. ok
t/ppi_node.t ..................... ok
t/28_foreach_qw.t ................ ok
t/24_v6.t ........................ ok
t/04_element.t ................... ok
t/interactive.t .................. ok
t/ppi_normal.t ................... ok
t/ppi_element.t .................. ok
t/ppi_lexer.t .................... ok
t/ppi_statement_include.t ........ ok
t/ppi_statement.t ................ ok
t/ppi_statement_compound.t ....... ok
t/ppi_statement_variable.t ....... ok
t/ppi_statement_sub.t ............ ok
t/ppi_statement_package.t ........ ok
t/ppi_token_dashedword.t ......... ok
t/ppi_token_number_version.t ..... ok
t/ppi_statement_scheduled.t ...... ok
t/ppi_token_magic.t .............. ok
t/ppi_token__quoteengine_full.t .. ok
t/ppi_token_quote.t .............. ok
t/ppi_token_quote_interpolate.t .. ok
t/ppi_token_quote_literal.t ...... ok
t/ppi_token_quote_double.t ....... ok
t/ppi_token_operator.t ........... ok
t/ppi_token_quotelike_words.t .... ok
t/ppi_token_quote_single.t ....... ok
t/ppi_token_prototype.t .......... ok
t/ppi_token_word.t ............... ok
t/21_exhaustive.t ................ ok
t/06_round_trip.t ................ ok

Test Summary Report
-------------------
t/07_token.t                   (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 447 tests but ran 1.
t/08_regression.t              (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 932 tests but ran 1.
t/05_lexer.t                   (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 219 tests but ran 1.
t/19_selftesting.t             (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
t/26_bom.t                     (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 21 tests but ran 1.
t/25_increment.t               (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 3876 tests but ran 1.
Files=52, Tests=8218,  8 wallclock secs ( 1.41 usr  0.23 sys + 18.61 cusr  1.73 csys = 21.98 CPU)
Result: FAIL

```

**Comment by plicease** [comment]
I believe this to be fixed by #196  and all that needs to be done is to do a release.

**Comment by wchristian** [comment]
It is fixed and released. :)
